frontend/packages/dev-console/src/test/pipeline-data.ts

```diff
@@ -1597,6 +1598,91 @@ export const pipelineTestData: PipelineTestData = {
       },
     },
   },
+  [PipelineExampleNames.PIPELINE_WITH_FINALLY]: {
+    dataSource: 'finally-pipeline',
+    pipeline: {
+      apiVersion: 'tekton.dev/v1alpha1',
+      kind: 'Pipeline',
+      metadata: {
+        name: 'finally-pipeline',
+        namespace: 'tekton-pipelines',
+      },
+      spec: {
+        tasks: [
+          {
+            name: 'hello-world-1',
+            taskRef: { name: 'hello-world-1' },
+          },
+        ],
+        finally: [
+          {
+            name: 'run-anyway',
+            taskRef: { name: 'run-anyway' },
+          },
+        ],
+      },
+    },
+    pipelineRuns: {
+      [DataState.SUCCESS]: {
+        apiVersion: 'tekton.dev/v1alpha1',
+        kind: 'PipelineRun',
+        metadata: {
+          name: 'finally-pipeline-3tt7aw',
+          namespace: 'tekton-pipelines',
+          labels: { [TektonResourceLabel.pipeline]: 'finally-pipeline' },
+        },
+        spec: {
+          pipelineRef: { name: 'finally-pipeline' },
+        },
+        status: {
+          pipelineSpec: {
```

Comment:
Since we do not have this `pipelineSpec` type defined in the `pipelineRunKind.status` in 4.6, there is a type error in this file as well.

Reply:
Updated.